syft/pkg/cataloger/javascript/options.go

```diff
@@ -0,0 +1,31 @@
+package javascript
+
+type CatalogerConfig struct {
+	searchRemoteLicenses bool
+	npmBaseURL           string
+}
+
+func (g CatalogerConfig) WithSearchRemoteLicenses(input bool) CatalogerConfig {
+	g.searchRemoteLicenses = input
+	return g
+}
+
+// NewCatalogerOpts create a NewCatalogerOpts with default options, which includes:
+// - searchRemoteLicenses is false
+func NewCatalogerOpts() CatalogerConfig {
+	g := CatalogerConfig{}
+
+	return g
+}
```

Comment:
I think this function should get removed in favor of consumers using the `DefaultCatalogerConfig()` below, which is consistent with the other cataloger configs, e.g. `javaCataloger.DefaultArchiveCatalogerConfig()` and `golangCataloger.DefaultCatalogerConfig()`